lib/histogram.js

```diff
@@ -280,12 +280,13 @@ function convertLabelsAndValues(labels, value) {
 function extractBucketValuesForExport(histogram) {
 	return bucketData => {
 		const buckets = [];
+		const bucketLabels = Object.entries(bucketData.labels);
```

Comment:
`Object.entries` is not available in Node 6 :(

Reply:
Whoops! I'll switch back to just shifting Object.keys up. Sorry!

Reply:
no worries! I thought we had the linter set up to yell, apparently not...

Reply:
#221 fwiw 🙂